johnsonandjohnson/Bodiless-JS
# feat(components): add a go to link for bodiless links in edit mode.

### Changes:
- Add a link to allow user to navigate to the link location in edit mode.

### Test Instructions:
- Add/edit/make a link
- Note there is a 'go' button
- Add a url in the link field.
- note the go button is active.
- on click the got button should take the user to the target location. 

### Fixes:
- #585
